ossdbtoolsservice/language/completion/pg_completer.py

```diff
@@ -12,20 +12,21 @@
 # {{ PGToolsService EDIT }}
 # from pgspecial.namedqueries import NamedQueries
 from prompt_toolkit.completion import Completer, Completion
+from ossdbtoolsservice.language.completion.packages.prioritization import PrevalenceCounter
 
 # {{ PGToolsService EDIT }}
 # from prompt_toolkit.document import Document
 # {{ PGToolsService EDIT }}
 from ossdbtoolsservice.language.completion.pg_completion import PGCompletion
-from .packages.sqlcompletion import (   # noqa
-    FromClauseItem, suggest_type, Database, Schema, Table, Function, Column, View,
-    Keyword, NamedQuery, Datatype, Alias, Path, JoinCondition, Join
-)
-from .packages.parseutils.meta import ColumnMetadata, ForeignKey
+
+from .packages.parseutils.pg_utils.meta import ColumnMetadata, ForeignKey
+from .packages.parseutils.pg_utils.tables import TableReference
 from .packages.parseutils.utils import last_word
-from .packages.parseutils.tables import TableReference
 from .packages.pgliterals.main import get_literals
-from .completer import MyCompleter, _Candidate, Match
```

Comment:
Can we check this file and add only required changes for PG. No additional refactoring required in this.